templates/zerver/help/send-a-private-message.md

```diff
@@ -2,31 +2,36 @@
 
 Zulip allows users to send private messages to each other.
 
-## There are two ways to send a Private Message:
+## You can send a private message by using the compose box or the users list.
+### Sending a private message through the compose box
 
-### Way 1:
+1. Click on the **New Private Message** button located at the bottom of your
+screen. The message box, like the one shown below will appear at the bottom
+center of the screen.
+  ![New stream message](/static/images/help/private-box.png)
 
-* To submit a new private message, first click on the `New Private
-  Message` button located at the bottom of your screen.
+2. In the field labeled **one or more people...**, input the username or email
+address of the user you want to privately message. When you begin typing an
+email address or username, autocomplete suggestions will appear.
 
-* The following box will be displayed
-![New stream message](/static/images/help/private-box.png)
+    !!! warn ""
+        If you attempt to input a nonexistent username or email address, an
+        error message will appear and the message will not be delivered.
 
-* In the **one or more people...** field, input the username or email
-address of the user you want to privately message.  When you begin
-typing an email address or username, autocomplete suggestions will
-appear.
+3. Enter your message in the box labeled **Compose your message here...**. You
+can now send your private message.
 
-* If you attempt to input a nonexistent username or email address, an
-  error message will appear.
+### Sending a private message through the user list.
 
-### Way 2:
+1. You also can click on the **user's name** in the user list on the right
+sidebar on the right and click on **down arrow** that gets visible when hovering
```

Comment:
Check the documentation styling guide for the down chevron icon code to replace `**down arrow**`